spec.md

```diff
@@ -201,6 +205,10 @@ A useful diagram is provided [here](https://github.com/google/go-containerregist
 A registry MAY reject a manifest of any type uploaded to the manifest endpoint if it references manifests or blobs that do not exist in the registry.
 When a manifest is rejected for this reason, it must result in one or more `MANIFEST_BLOB_UNKNOWN` errors <sup>[code-1](#error-codes)</sup>.
 
+The client SHOULD include a `Docker-Manifest-Tag` header indicating which manifest tag is being pushed.
+This header SHOULD be added to all requests throughout the push process.
+Registries MAY use this value as a hint when handling the pushed manifests and blobs.
```

Comment:
Similar to pull, consider dropping this sentence since "hint" is too abstract.

Reply:
OK, will drop it.